kclvm/runner/src/test_datas/multi_file_compilation/large_scale_case/main.k

```diff
@@ -0,0 +1,49 @@
+import base.pkg.kusion_models.kube.frontend
```

Comment:
The test suite is too large. Considering Konfig integration testing directly through git sub module

Reply:
fixed